coop_point_of_sale/models/pos_session.py

```diff
@@ -29,6 +29,26 @@ class PosSession(models.Model):
         string='Day (Search)', compute='_compute_date_search',
         multi='_date_search', store=True, index=True)
 
+    order_qty = fields.Integer(
+        compute='_compute_orders', string='Orders Qty', store=True)
+
+    amount_subtotal = fields.Monetary(
+        compute='_compute_orders', string='Amount Subtotal', store=True)
+
+    total_amount = fields.Monetary(
+        compute='_compute_orders', string='Transactions Total', store=True)
+
+
+    @api.multi
+    @api.depends('order_ids.lines.price_subtotal_incl', 'order_ids.lines.price_subtotal')
```

Comment:
```suggestion
    @api.depends('order_ids.amount_total', 'order_ids.amount_tax')
```
I don't get why depending on the line value :thinking: 
This should be enough and it's clearer